### DISCUSSION TITLE
TOC and page content overlap

### DISCUSSION THREAD

**Discussion by lucamondini** [jpanther/congo#743] [Q&A] [answered]
Hi all, I noticed that the TOC is now overlapping with the page content, and I'm pretty sure when I published them there was no problem at all.
Example [here](https://www.lucamondini.it/cv/).
I found [this post](https://github.com/jpanther/congo/issues/576) that describes the exact same issue but everything is up to date on my side.
Do you have any ideas about the possible cause?

Thanks in advance :)

Environment:
- Hugo 0.121.0
- Congo 2.7.6
- Browser Safari / Chrome

**Comment by GorkemGuray** [comment, marked as answer]
Hi,
Is there a layout you override? I was overriding the single.html file before, when I updated the theme I encountered the same problem. I think you should check your layout files.

**Comment by wolfspyre** [comment]
heya! 
Look in your config for "-ms-5":

![congo-743_1](https://github.com/jpanther/congo/assets/1680659/d8267096-2d23-48e8-8cfc-58c52e8bd0cb)
![congo-743_2](https://github.com/jpanther/congo/assets/1680659/af43c738-b30d-4b0a-89f4-fb51a74012a9)

**Comment by lucamondini** [comment]
Thank you both, there was an outdated layout override. Problem solved!
